src/ImageSharp/Formats/Webp/Lossless/HistogramEncoder.cs

```diff
@@ -50,18 +61,17 @@ public static void GetHistoImageSymbols(int xSize, int ySize, Vp8LBackwardRefs r
         bool entropyCombine = numUsed > entropyCombineNumBins * 2 && quality < 100;
         if (entropyCombine)
         {
-            ushort[] binMap = mapTmp;
             int numClusters = numUsed;
             double combineCostFactor = GetCombineCostFactor(imageHistoRawSize, quality);
-            HistogramAnalyzeEntropyBin(imageHisto, binMap);
+            HistogramAnalyzeEntropyBin(imageHisto, mapTmp);
 
             // Collapse histograms with similar entropy.
-            HistogramCombineEntropyBin(imageHisto, histogramSymbols, clusterMappings, tmpHisto, binMap, entropyCombineNumBins, combineCostFactor);
+            HistogramCombineEntropyBin(imageHisto, histogramSymbols, clusterMappings, tmpHisto, mapTmp, entropyCombineNumBins, combineCostFactor);
 
             OptimizeHistogramSymbols(clusterMappings, numClusters, mapTmp, histogramSymbols);
         }
 
-        float x = quality / 100.0f;
+        float x = quality / 100F;
```

Comment:
```suggestion
        float x = quality * 0.01F;
```
Division is more expensive than multiplication.

Reply:
I'd need to check again if this is on a hot path.